package.json

```diff
@@ -19,18 +19,20 @@
     "build": "bsb -make-world",
     "clean": "bsb -clean-world",
     "start": "bsb -make-world -w",
-    "pretest": "rimraf tests/main.bs.js && yarn build && sed -i '1s;^;require(\"@babel/polyfill\");\n' tests/main.bs.js",
-    "test": "browserify tests/main.bs.js | tape-run --browser chrome --render=faucet"
+    "pretest": "rimraf tests/main.bs.js && yarn build && prepend tests/main.bs.js 'require(\"abortcontroller-polyfill/dist/abortcontroller-polyfill-only\");require(\"@babel/polyfill\");\n'",
```

Comment:
Much closer. To get it working I had to change `pretest` to:
```javascript
rimraf tests/main.bs.js && yarn build && prepend tests/main.bs.js \"require('abortcontroller-polyfill/dist/abortcontroller-polyfill-only');require('@babel/polyfill');\n\"
```
The quotes were breaking things.